lib/bootstrap_form/form_builder.rb

```diff
@@ -61,9 +61,18 @@ def initialize(object_name, object, template, options)
 
     def file_field_with_bootstrap(name, options = {})
       prevent_prepend_and_append!(options)
-      options = options.reverse_merge(control_class: 'form-control-file')
+      options = options.reverse_merge(control_class: "custom-file-input")
       form_group_builder(name, options) do
-        file_field_without_bootstrap(name, options)
+        content_tag(:div, :class => "custom-file") do
```

Comment:
So we allow the programmer to use only the Bootstrap 4 custom file dialog, and not the browser's built-in dialog?

Reply:
To clarify: the "custom" part is how the input is styled. When the user clicks on the "browse" button, they still get the browser's built-in dialog. So I think this is OK.